e2e/rbd.go

```diff
@@ -2666,6 +2666,10 @@ var _ = Describe("RBD", func() {
 				if err != nil {
 					e2elog.Failf("failed to delete PVC and application: %v", err)
 				}
+				err = deleteJournalInfoInPool(f, pvc, "replicapool")
```

Comment:
can these _leaks_ be fixed otherwise? do they also leak in normal/production environments?

Reply:
I will raise a new issue for this, let's take it there.

Reply:
Did you do this? Please comment here with links to the issue(s).

Reply:
seems to be #3111